test/distributed/org/apache/cassandra/fuzz/harry/integration/dsl/HistoryBuilderIntegrationTest.java

```diff
@@ -19,46 +19,31 @@
 package org.apache.cassandra.fuzz.harry.integration.dsl;
 
 import java.util.Random;
-import java.util.function.LongSupplier;
 import java.util.function.Supplier;
 
 import org.junit.Test;
 
-import org.apache.cassandra.fuzz.harry.integration.model.ModelTestBase;
+import org.apache.cassandra.fuzz.harry.integration.model.IntegrationTestBase;
 import org.apache.cassandra.harry.checker.ModelChecker;
-import org.apache.cassandra.harry.core.Configuration;
-import org.apache.cassandra.harry.core.Run;
 import org.apache.cassandra.harry.ddl.SchemaGenerators;
 import org.apache.cassandra.harry.ddl.SchemaSpec;
 import org.apache.cassandra.harry.dsl.BatchVisitBuilder;
 import org.apache.cassandra.harry.dsl.HistoryBuilder;
 import org.apache.cassandra.harry.dsl.SingleOperationBuilder;
-import org.apache.cassandra.harry.dsl.ValueDescriptorIndexGenerator;
 import org.apache.cassandra.harry.gen.rng.JdkRandomEntropySource;
 import org.apache.cassandra.harry.model.Model;
 import org.apache.cassandra.harry.operations.Query;
 import org.apache.cassandra.harry.sut.SystemUnderTest;
 import org.apache.cassandra.harry.sut.TokenPlacementModel;
+import org.apache.cassandra.harry.tracker.DataTracker;
+import org.apache.cassandra.harry.tracker.DefaultDataTracker;
 import org.apache.cassandra.harry.visitors.ReplayingVisitor;
 
-public class HistoryBuilderIntegrationTest extends ModelTestBase
+public class HistoryBuilderIntegrationTest extends IntegrationTestBase
 {
     private final long seed = 1L;
     private final int STEPS_PER_ITERATION = 1_000;
-    private final int MAX_PARTITIONS = 100;
-
-    @Override
-    protected Configuration.ModelConfiguration modelConfiguration()
-    {
-        return new Configuration.QuiescentCheckerConfig();
-    }
-
-    public Configuration.ConfigurationBuilder configuration(long seed, SchemaSpec schema)
-    {
-        return super.configuration(seed, schema)
-                    .setPartitionDescriptorSelector(new Configuration.DefaultPDSelectorConfiguration(2, 2))
-                    .setClusteringDescriptorSelector((builder) -> builder.setOperationsPerLtsDistribution(new Configuration.ConstantDistributionConfig(100_000)));
-    }
+    private final int MAX_PARTITIONS = 50;
```

Comment:
nit: I guess `MAX_PARTITIONS` and `STEPS_PER_ITERATION` could be static just to avoid annoying IDE warnings, but not that important...

Reply:
Did that for some of the fields I have found.